libs/core/langchain_core/output_parsers/openai_tools.py

```diff
@@ -52,7 +66,15 @@ def make_invalid_tool_call(
     raw_tool_call: Dict[str, Any],
     error_msg: Optional[str],
 ) -> InvalidToolCall:
-    """Create an InvalidToolCall from a raw tool call."""
+    """Create an InvalidToolCall from a raw tool call.
+
+    Args:
+        raw_tool_call: The raw tool call.
```

Comment:
Some of the doc-strings are not providing any information that's not already present in the function signature. Does this help to improve the API reference by naming arguments?

Reply:
IMHO It is kind of the "maturity" index. If I see this, it it 100% missed content. This content can be 100% trivial and uninformative. 
We have a list of Args, dashes that tell reader "here is a place for description", and this missed description.
![image](https://github.com/langchain-ai/langchain/assets/2256422/ac05f3ac-271c-4c5b-8343-3f3add3a4443)